docs/guides/pagination/offset-based.md

```diff
@@ -134,7 +134,7 @@ The return type contains additional metadata that the client can query for, to k
 
 For simplicity's sake, our dataset is going to be an in-memory list.
 
-```py line=7-32
+```py line=7-31
```

Comment:
I believe this should still be 32
![image](https://user-images.githubusercontent.com/70145864/150523004-8e56283c-9006-495e-a96b-7d16701de93a.png)

Reply:
Yes, you're right! I'll change it ASAP!